akvo/rest/views/typeahead.py

```diff
@@ -59,7 +59,15 @@ def typeahead_user_organisations(request):
 
 @api_view(['GET'])
 def typeahead_project(request):
-    projects = Project.objects.all().exclude(title='')
+    from akvo.rsr.views.project import _project_directory_coll
+    if request.GET.get('published', '0') == '0':
```

Comment:
Shouldn't the projects be filtered by employment and group in the project editor? So that you don't get info on projects you can't access anyway. This becomes important if we expand the scope of the data returned by the typeahead I think.